File: atest.lua
#!/usr/bin/env lua





io.write("Write Something:\n")

for l in io.lines() do


	io.write("You typed: ", l, "\n")
	if l == "q" then
		break
	end
	io.write("Write Something:\n")
end
io.write("Quit Program\n")

File: atest.erl
#!/usr/bin/env escript
-module(atest).

main(_) ->
	processLines(),
	io:format("Quit Program\n").


processLines() ->
	Line = io:get_line("Write Something:\n"),

	io:format("You typed: ~s", [Line]),
	case Line of
		"q\n" -> ok;
		_     -> processLines()
	end.

